auslib/web/admin/swagger/api.yaml

```diff
@@ -108,6 +131,110 @@ parameters:
     minimum: 0
     required: true
 
+  change_id_param:
+    name: change_id
+    in: path
+    description: Unique id to identify the identify the change
+    type: string
```

Comment:
That seems like a bug. It's definitely an integer in the database (https://github.com/mozilla/balrog/blob/2a245b6e25ff38a4b896fb6965d66d884bb9d79a/auslib/db.py#L629), and it's returned by the /rules/:rule_id/revisions endpoint as an integer. Can you make this an integer, too?